src/gui/mzroll/mainwindow.cpp

```diff
@@ -2458,6 +2465,29 @@ int MainWindow::loadMetaCsvFile(string filename){
 
         }
 
+        qreal *redF, *greenF, *blueF, *alpha;
+        if (header.count("color") && header["color"] < N) {
+
+            string colorName = fields[header["color"]];
+            redF = new qreal(-1);
+            greenF = new qreal(-1);
+            blueF = new qreal(-1);
+            alpha = new qreal(-1);
```

Comment:
@saifulbkhan  I have changed this pointer implementation. Actually, 'getRgbF' required these values as pointers, I have passed addresses instead of making them pointers